src/sentry/api/serializers/models/sentry_app_installation.py

```diff
@@ -19,7 +19,7 @@ def serialize(self, install, attrs, user):
             'uuid': install.uuid,
         }
 
-        if install.is_new:
```

Comment:
if we are removing this, can we remove where it gets set/the property itself? Or do we/would we use this somewhere else?

Reply:
Actually, I should probably use this elsewhere in addition to checking whether the grant exists. I'll keep it for that and make the change.